.github/workflows/release-please.yml

```diff
@@ -1,79 +1,55 @@
+name: Release
 on:
   push:
     branches:
       - master
-name: release-please
+permissions:
+  contents: write
+  pull-requests: write
 jobs:
   release-please:
+    name: Release Please
     runs-on: ubuntu-latest
-    outputs:
-      release_created: ${{ steps.release.outputs.release_created }}
-      sha: ${{ steps.release.outputs.sha }}
-      upload_url: ${{ steps.release.outputs.upload_url }}
-    # https://github.com/google-github-actions/release-please-action in case more config is needed
     steps:
       - uses: google-github-actions/release-please-action@v3
         id: release
         with:
+          release-type: node
```

Comment:
I believe this info gets configured in the command: manifest anyway?

Reply:
That strange because its never had issues without it so its "requirement" seems questionable :thinking: 

Oddly their main repo https://github.com/google-github-actions/release-please-action

Just suggests the lines without the `release-type: node` when using the manifests.

Reply:
I think this should be a problem with Google, I will delete this line.

Maybe I should submit an issue to google-github-actions/release-please-action.